integration-tests/src/test/java/oracle/kubernetes/operator/ItJrfModelInImage.java

```diff
@@ -0,0 +1,242 @@
+    assertDoesNotThrow(() -> createResultAndPvDirs(testClassName),
+        "Failed: createResultAndPvDirs");
+    
+    LoggerHelper.getLocal().log(Level.INFO, "Copying sample dir to the result dir");
+    assertDoesNotThrow(() -> TestUtils.exec(
+        "cp -rf " 
+         + BaseTest.getProjectRoot() 
+         + "/kubernetes/samples/scripts " 
+         + getResultDir(),
+         true),
+         "Failed: Copy sample dir to the result dir");
+    
+    //start DB and create RCU
+    dbNamespace = "db" + String.valueOf(getNewSuffixCount());
+    assertDoesNotThrow(() -> DbUtils.createNamespace(dbNamespace),
+        "Failed to create namespace: " + dbNamespace);
+    dbPort = 30011 + getNewSuffixCount();
+    dbUrl = "oracle-db." + dbNamespace + ".svc.cluster.local:1521/devpdb.k8s";
+    assertDoesNotThrow(() -> DbUtils.createDbRcu(getResultDir(), dbPort, dbUrl, 
+        rcuSchemaPrefix, dbNamespace));
+    LoggerHelper.getLocal().log(Level.INFO,"RCU schema is created for test: " 
+        + testClassName 
+        + " dbNamespace is: " + dbNamespace 
+        + " dbUrl:" + dbUrl 
+        + " dbPort: " + dbPort
+        + " rcuSchemaPrefix" + rcuSchemaPrefix); 
+        
+    // create operator
+    if (operator == null) {
+      Map<String, Object> operatorMap = createOperatorMap(getNewSuffixCount(),
+          true, testClassName);
+      operator = assertDoesNotThrow(() -> TestUtils.createOperator(operatorMap, RestCertType.SELF_SIGNED));
+      assertNotNull(operator);
+      LoggerHelper.getLocal().log(Level.INFO, "Operator is created for {0}", testClassName);
+      
+      domainNS = ((ArrayList<String>) operatorMap.get("domainNamespaces")).get(0);
+      namespaceList.append((String)operatorMap.get("namespace"));
+      namespaceList.append(" ").append(domainNS);
+    }
+  }
+  
+  /**
+   * This method will run once after all test methods are finished. It delete both RCU and DB pods
+   */
+  @AfterEach
+  public void unPrepare() {
+    LoggerHelper.getLocal().log(Level.INFO, "Is going to drop RCU schema and stop DB for {0}", testClassName);
+    assertDoesNotThrow(() -> DbUtils.deleteRcuPod(getResultDir()),
+        "Failed: drop RCU schema");
+    assertDoesNotThrow(() -> DbUtils.deleteDbPod(getResultDir()),
+        "Failed: stop DB");
+  }
+
+  /**
+   * Releases k8s cluster lease, archives result, pv directories.
+   */
+  @AfterAll
+  public static void staticUnPrepare() {
+    assertDoesNotThrow(() -> tearDown(new Object() {
+        }.getClass().getEnclosingClass().getSimpleName(), namespaceList.toString()),
+         "tearDown failed");
+  }
+
+  /**
+   * Create and deploy a JRF domain using model in image. Save and restore walletFileSecret. After shutting down 
+   * the domain, enable walletFileSecret in the domain yaml file. Restart the domain with the modified domain yaml 
+   * file that reuses the same RCU schema. Verify the restarted domain is in the good state.
```

Comment:
Please qualify good state, maybe we can say domain is re-stated successfully with all server pod up and running

Reply:
Done